adr/005-oesa.md

```diff
@@ -22,7 +22,27 @@ There are a couple main concepts that we strive to maintain while also being pra
    less important code depending on more important code.  For example, when business logic needs to call the
    database.
 
-### Related Issues
+## Impact
```

Comment:
This comment might be out of scope for this particular PR, but when I first started, I couldn't even figure out what OeSA _was_. It would be really helpful to add some more context on it (at the very least, that it means 'option-enabling software architecture', plus probably that it's a Flexion thing, and since the Flexion info is all non-public, maybe like a two sentence summary of what it is

Reply:
I don't think it's specifically a Flexion-thing per say? I know it employs a lot of the Clean Coding practices from the famous Uncle Bob. I need to verify if OeSA is even still the name, but when/if correct - will add that. I think the concept may have evolved a bit since then as the content is being honed. 

@halprin @sfradkin , maybe you can correct me if I'm completely out of order?

Reply:
>  It would be really helpful to add some more context on it (at the very least, that it means 'option-enabling software architecture'...

Also forgot - the title shows the full acronym already. Is that okay?

Reply:
I made a chunk of edits to the OEA adr. Open to suggestions after a re-review.